pkg/apis/pingcap/v1alpha1/defaulting/tidbcluster.go

```diff
@@ -26,6 +26,11 @@ const (
 	defaultBinlogImage = "pingcap/tidb-binlog"
 )
 
+var (
+	tidbLogMaxSize    = 300
```

Comment:
`300` means `300MB`? I think we don't need to set it if it's the default value of the `max-size`.

Reply:
So we only need set the default max-page? I don't know whether 3 is proper for the default `max-backups` in kubernetes cases.

Reply:
yes

there is no perfect value because ephemeral storage is not the same in different environments. I think `3` is ok, about 900 MiB size.